File: DAY_0_FOUNDATION/Triangle_Type_by_Angles.py
Angle1=int(input("Enter the first angle: "))
Angle2=int(input("Enter the second angle: "))
Angle3=int(input("Enter the third angle: "))
if Angle1>0 and Angle1>0 and Angle3>0:
    if Angle1<90 and Angle2<90 and Angle3<90:
        print("The angle is Acute !!")
    elif Angle1>90 or Angle2>90 or Angle3>90:
        print("The angle is Obtuse !!")
    else:
        print("The angle is Right Angle !!")
else:
    print("Angles are not valid !!")

File: 3.CONDITIONAL STATEMENTS (IF-ELSE)2/Triangle_Type_by_Angles.cpp
#include <iostream>
using namespace std;

int main() {
    int angle1, angle2, angle3;


    cout << "Enter first angle: ";
    cin >> angle1;
    cout << "Enter second angle: ";
    cin >> angle2;
    cout << "Enter third angle: ";
    cin >> angle3;

    int sum = angle1 + angle2 + angle3;

    if (sum != 180 || angle1 <= 0 || angle2 <= 0 || angle3 <= 0) {
        cout << "Invalid Triangle" << endl;
    } else {
        
        if (angle1 < 90 && angle2 < 90 && angle3 < 90) {
            cout << "Acute Triangle" << endl;
        } else if (angle1 == 90 || angle2 == 90 || angle3 == 90) {
            cout << "Right Triangle" << endl;
        } else { // One angle > 90
            cout << "Obtuse Triangle" << endl;
        }
    }

    return 0;
}
